test/c_api/flat_tuple_test.cpp

```diff
@@ -65,7 +65,7 @@ TEST_F(CApiFlatTupleTest, ToString) {
     columnWidths[1] = 5;
     columnWidths[2] = 10;
     char* str = kuzu_flat_tuple_to_string(&flatTuple);
-    ASSERT_EQ(std::string(str), "Alice|35|1.731000\n");
+    ASSERT_EQ(std::string(str), "Alice|35|1.731\n");
```

Comment:
Instead of going through all the test files and updating the precision of floating point numbers, we should have a smarter testing framework which is similar to pytest comparison:
```
assert 2.2 == pytest.approx(2.3, 0.1)
```
You can leave this part to us.